servers/audio_server.cpp

```diff
@@ -1230,6 +1230,7 @@ void AudioServer::start_playback_stream(Ref<AudioStreamPlayback> p_playback, con
 	int idx = 0;
 	for (KeyValue<StringName, Vector<AudioFrame>> pair : p_bus_volumes) {
 		if (pair.value.size() < channel_count || pair.value.size() != MAX_CHANNELS_PER_BUS) {
+			delete playback_node;
 			delete new_bus_details;
```

Comment:
Is there any reason why `AudiServer` is using generic `new`/`delete` instead of our custom `memnew`/`memdelete`?